tools/spelling/spelling_dictionary.txt

```diff
@@ -1537,3 +1537,4 @@ NAT
 NXDOMAIN
 DNAT
 RSP
+FFFD
```

Comment:
do you still need this? It looks like you are backticking the comment correctly.

Reply:
Yeah, the format CI still complain.

Reply:
Complains on what line? It's going to be pretty useless to have a spelling checker if we have to allow-list a bunch of random hex codes.

Reply:
I will paste the error it here